pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -591,7 +592,7 @@ func (r *PerformanceProfileReconciler) reconcileCgroupsV1(ctx context.Context, i
 	}
 
 	// (TODO) This code can be removed in the future when the cgroupsv2 is supported
-	currentMC, err := r.getCurrentMachineConfigByMCP(ctx, profileMCP)
```

Comment:
we can remove the getCurrentMachineConfigByMCP function if its not used at all (linter complaints about it anyway)

Reply:
done